### PR TITLE
[SYCL][L0] Enable round-robin submissions to multiple compute CCS

### PR DESCRIPTION
**Pull request by smaslov-intel**

### CODE REVIEW

File: sycl/plugins/level_zero/pi_level_zero.cpp [hunk 1]
```diff
@@ -342,6 +342,26 @@ class ReturnHelper {
 
 } // anonymous namespace
 
+// SYCL_PI_LEVEL_ZERO_USE_COMPUTE_ENGINE can be set to an integer (>=0) in
```
Comment (asudarsa) [inline sycl/plugins/level_zero/pi_level_zero.cpp:345]: Maybe e can follow the same format as that for copy engine? 'lower_index:upper_index ?

Comment (smaslov-intel) [reply sycl/plugins/level_zero/pi_level_zero.cpp:345]: I didn't see value in it, but it can be added later if needed.

**Comment by asudarsa** [comment]
Please run perf testing (atleast SYCL benchmarks) before submission.

**Review by asudarsa — APPROVED** [review]
LGTM. Thanks

**Review by againull — APPROVED** [review]

**Pull request merged**